views/index.ejs

```diff
@@ -8,9 +8,12 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length %> </strong>
     <ul>
-      <!-- List the names of the items in your cart here -->
+      <% cart.forEach(function(product) { %>
+        <li><%= product.name %></li>
+      <% }) %>
+
```

Comment:
You've got your forEach loop set up correctly, but now you need some way to access the product's id from this form so your post route has the ability to get the id from req.body.id. Since you don't necessarily want the id to print by each cookie, you can have a hidden input with the value set to `<%= product.id %>`.